frontend/src/App.js

```diff
@@ -1,21 +1,19 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Routes from './routes/Routes.js';
```

Comment:
Basically you can remove extension from the import.

http://discuss.babeljs.io/t/es6-import-jsx-without-suffix/172

I think `create-react-app` already includes this preset.